aws-cpp-sdk-core/CMakeLists.txt

```diff
@@ -402,6 +397,13 @@ else()
     endif()
 endif()
 
+configure_file("${CMAKE_SOURCE_DIR}/toolchains/pkg-config.pc.in" "${PROJECT_NAME}.pc" @ONLY)
```

Comment:
Just based on the diff, the configure_file was on line 378 and then it got moved after the "if(SIMPLE_INSTALL)" block to line 400.

Similarly, the "if(BUILD_SHARED_LIBS)" block within "if(SIMPLE_INSTALL)" got moved from line 388 to line 401.

Should all CMakeLists.txt have the same change applied to them? or was this just an harmless accidental change that doesn't actually change anything?